samples/python/reactors/PorousMediaBurner.py

```diff
@@ -0,0 +1,395 @@
+"""
+Short description: reactor cascade model for reactive flows in inert porous media
+based on extensible reactors. Showcases adding a temperature equation for a
+solid-phase and custom heat transfer/radiation models.
+
+Code by Thorsten Zirwes and Guillaume Vignat
+Stanford University & Karlsruhe Institute of Technology (KIT)
+2023
+
+This code implements a reactor cascade model for the simulation of reactive
```

Comment:
Please add a disclaimer stating limitations of the model, as is uses drastic simplifications in order to convert it from a 1D problem to a 0D cascade.